Additional fields to access log messages

aravindavk opened the pull request:
Added Method, URL, Response code and duration to the log message. Changed request log message to debug to avoid two log messages for one request in production.

Example log:

```
...y_httpd_dist-worker-1 INFO  2024-07-02T22:55:42.419 Response: GET /api/v1/volumes - 200 OK (53 ms, 977 μs)
```

andrewlalis commented:
Looks good; one thing though: I don't want the default behavior to permanently write remote addresses to logs, since that can be considered sensitive information according to some regulations probably.

aravindavk commented:
> Looks good; one thing though: I don't want the default behavior to permanently write remote addresses to logs, since that can be considered sensitive information according to some regulations probably.

Makes sense. Removed the remote address and updated the PR.
